nautobot/docs/configuration/optional-settings.md

```diff
@@ -724,6 +728,8 @@ The amount of time (in seconds) to wait for NAPALM to connect to a device.
 
 Default: `~/.nautobot/`
 
+Environment Variable: `NAUTOBOT_ROOT`
+
```

Comment:
LIkewise here, environment variable `$NAUTOBOT_ROOT` is the only relevant setting - `settings.NAUTOBOT_ROOT` does nothing. 

Maybe instead of adding `Environment Variable:` entries to GIT_SSL_NO_VERIFY and NAUTOBOT_ROOT, we should move these environment-variable-only settings to a new, separate section of this document?